website/docs/intro/install/alpine-convenience.sh

```diff
@@ -1,6 +1,6 @@
 # Download the installer script:
-curl --proto '=https' --tlsv1.2 -fsSL https://get.opentofu.org/install-opentofu.sh -o install-opentofu.sh
-# Alternatively: wget --secure-protocol=TLSv1_2 --https-only https://get.opentofu.org/install-opentofu.sh -o install-opentofu.sh
+curl --proto '=https' --tlsv1.2 -fsSL https://get.opentofu.org/install-opentofu.sh -O install-opentofu.sh
```

Comment:
This is curl, not wget.  Wget is what requires -O

Reply:
Oops, fixed!